.tekton/tasks/verify-smoke-e2e.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: stage-verify-e2e-test
+spec:
+  description: >-
+    This task is for running stage e2e test on RHTAP staging cluster
+  params:
+  - name: PLATFORM
+    description: which environment the test is running on.
+    default: "stage"
+  - name: TASK_RETRIES
+    description: number of times the task will be rerun
+    default: "0"
+  workspaces:
+    - name: stage-e2e-git
+  steps:
+    - name: run-stage-verify-e2e-test
+      image: registry.access.redhat.com/ubi8/go-toolset:1.19.10-10.1692783630
+      workingDir: $(workspaces.stage-e2e-git.path)
+      script: |
+        #!/usr/bin/env bash
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        mkdir -p ~/.kube
+        cat <<EOF > ~/.kube/config
+        apiVersion: v1
+        clusters:
+        - cluster:
+            server: YOUR_KUBE_API_SERVER_URL
+          name: YOUR_CLUSTER_NAME
+        contexts:
+        - context:
+            cluster: YOUR_CLUSTER_NAME
+            user: YOUR_USER_NAME
+          name: YOUR_CONTEXT_NAME
+        current-context: YOUR_CONTEXT_NAME
+        kind: Config
+        preferences: {}
+        users:
+        - name: YOUR_USER_NAME
+          user:
+            token: YOUR_KUBE_API_TOKEN
```

Comment:
do we have an estimation about how much time the tests takes? Ussually our rhtap-demo takes around 20m and if it is the sandbox token expiry in 15m

Reply:
We will be running smoke test. So it should be quick.

Reply:
What exactly are smoke tests?